### PR TITLE
change endpoint grouping/naming to be focused on objects

### PR DESCRIPTION
**Pull request by andrewhong5297**
edited sections to be "tables", "queries", and "executions and results" and moved around a few subpages.

### CODE REVIEW

File: api-reference/quickstart/queries-eg.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: 'CRUD Repo Setup'
+title: 'Query Management'
```
Comment (philippWassibauer) [inline api-reference/quickstart/queries-eg.mdx:2]: Shouldn't we add something about Repo here? eg: Query management with Github Repo?

File: mint.json [hunk 2]
```diff
@@ -179,42 +176,44 @@
     {
       "group": "API Quickstart",
       "pages": [
-        "api-reference/quickstart/query-eg",
-        "api-reference/quickstart/upload-eg",
-        "api-reference/quickstart/crud-eg"
+        "api-reference/quickstart/results-eg",
```
Comment (philippWassibauer) [inline mint.json:179]: what is `eg`?

**Review by philippWassibauer — APPROVED** [review]
just 2 comments. lfg, looks good

File: mint.json (hunk 2 above)
Comment (andrewhong5297) [reply mint.json:179]: I think it just means example? Eg, ie

File: api-reference/quickstart/queries-eg.mdx (hunk 1 above)
Comment (andrewhong5297) [reply api-reference/quickstart/queries-eg.mdx:2]: Ah yeah I had repo in there before but it was too long and made the left nav bar long haha

**Pull request merged**